css/selectors/marker.json

```diff
@@ -64,11 +64,16 @@
             "opera_android": {
               "version_added": "61"
             },
-            "safari": {
-              "version_added": "11.1",
-              "partial_implementation": true,
-              "notes": "Safari support is limited to <code>color</code> and <code>font-size</code>. See <a href='https://webkit.org/b/204163'>bug 204163</a>."
-            },
+            "safari": [
```

Comment:
So, this is an interesting case. Here, we'll want to put the preview statement second, since it's less informative than the second entry, which applies more broadly.